test/e2e_node/e2e_node_suite_test.go

```diff
@@ -101,7 +101,7 @@ func TestE2eNode(t *testing.T) {
 // Setup the kubelet on the node
 var _ = SynchronizedBeforeSuite(func() []byte {
 	if *buildServices {
-		buildGo()
+		Expect(buildGo()).To(Succeed())
```

Comment:
I assume the assertion failure won't cause the process to exit. Could you confirm that?